x-pack/plugin/esql/qa/testFixtures/src/main/resources/change_point.csv-spec

```diff
@@ -0,0 +1,1199 @@
+2024-05-10T00:20:00.000Z | 10         | null         | null                 
+2024-05-10T00:21:00.000Z | 4          | null         | null                 
+2024-05-10T00:22:00.000Z | 9          | null         | null                 
+null                     | 5          | null         | null  
+null                     | 12         | null         | null                 
+;
+
+null values
+required_capability: change_point
+
+FROM k8s
+  | STATS count=COUNT() BY @timestamp=BUCKET(@timestamp, 1 MINUTE)
+  | EVAL count=count+CASE(@timestamp>="2024-05-10T00:11:00.000Z", 100, 0)
+  | EVAL count=CASE(@timestamp=="2024-05-10T00:04:00.000Z", NULL, count)
+  | CHANGE_POINT count ON @timestamp AS type, pvalue
+;
+
+warning:Line 5:3: warnings during evaluation of [CHANGE_POINT count ON @timestamp AS type, pvalue]. Only first 20 failures recorded.
+warning:Line 5:3: java.lang.IllegalArgumentException: values contain nulls; skipping them
+
+@timestamp:datetime      | count:long | type:keyword | pvalue:double
+2024-05-10T00:00:00.000Z | 4          | null         | null
+2024-05-10T00:01:00.000Z | 4          | null         | null
+2024-05-10T00:02:00.000Z | 8          | null         | null
+2024-05-10T00:03:00.000Z | 8          | null         | null
+2024-05-10T00:04:00.000Z | null       | null         | null
+2024-05-10T00:05:00.000Z | 8          | null         | null
+2024-05-10T00:06:00.000Z | 10         | null         | null
+2024-05-10T00:07:00.000Z | 5          | null         | null
+2024-05-10T00:08:00.000Z | 12         | null         | null
+2024-05-10T00:09:00.000Z | 20         | null         | null
+2024-05-10T00:10:00.000Z | 5          | null         | null
+2024-05-10T00:11:00.000Z | 107        | step_change  | 3.438939970021414E-22
+2024-05-10T00:12:00.000Z | 108        | null         | null
+2024-05-10T00:13:00.000Z | 109        | null         | null
+2024-05-10T00:14:00.000Z | 109        | null         | null
+2024-05-10T00:15:00.000Z | 111        | null         | null
+2024-05-10T00:16:00.000Z | 107        | null         | null
+2024-05-10T00:17:00.000Z | 115        | null         | null
+2024-05-10T00:18:00.000Z | 117        | null         | null
+2024-05-10T00:19:00.000Z | 105        | null         | null
+2024-05-10T00:20:00.000Z | 110        | null         | null
+2024-05-10T00:21:00.000Z | 104        | null         | null
+2024-05-10T00:22:00.000Z | 109        | null         | null
+;
+
+
+multivalued
+required_capability: change_point
+
+FROM employees
+  | STATS salary=MV_SORT(VALUES(salary)) BY height
+  | EVAL salary = CASE(height == 1.5, [1, 22222, 33333], salary)
+  | EVAL salary = CASE(height == 1.63, [43210, -10000, 999999999], salary)
+  | CHANGE_POINT salary ON height
+;
+
+warning:Line 5:3: evaluation of [CHANGE_POINT salary ON height] failed, treating result as null. Only first 20 failures recorded.
+warning:Line 5:3: java.lang.IllegalArgumentException: values is multivalued; please reduce them with e.g. MV_AVG or MV_SUM
```

Comment:
Suggestion: warning makes it not clear to me that this will not just ignore multivalues and remove them from the values - which we do with nulls. I think we should be clearer on this.

In most of ESQL, MVs are simply ignored but they don't invalidate the whole command. (I'm not saying that CHANGE_POINT should just ignore MVs, like we ignore nulls. This probably has to be consistent with the change point detection already in place in ES.)